### PR TITLE
chore(session replay): bump sr sdk version from 0 to 1

### PR DESCRIPTION
**Pull request by jxiwang**
### Summary

Bumping version from v0.x -> v1.x for SR GA

### Checklist

* [ ] Does your PR title have the correct [title format](https://github.com/amplitude/Amplitude-TypeScript/blob/main/CONTRIBUTING.md#pr-commit-title-conventions)?
* Does your PR have a breaking change?:

### CODE REVIEW

File: packages/plugin-session-replay-browser/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@amplitude/plugin-session-replay-browser",
-  "version": "0.11.0-beta.0",
+  "version": "1.0.0-beta.0",
```
Comment (justin-fiedler) [inline packages/plugin-session-replay-browser/package.json:3]: Are you keeping this in beta? I think adding the dependency at `1.0.0` will not retrieve a beta version.

Comment (jxiwang) [reply packages/plugin-session-replay-browser/package.json:3]: Ah okay, so 1.0.0 is fine? I was going to pre-release then graduate but okay with 1.0.0 if that is correct procedure.

Comment (justin-fiedler) [reply packages/plugin-session-replay-browser/package.json]: I think its ok to do a prelease and then graduate. Just wasn't sure if you had kept beta intentionally

**Review by justin-fiedler — APPROVED** [review]

File: packages/plugin-session-replay-browser/package.json (hunk 1 above)
Comment (jxiwang) [reply packages/plugin-session-replay-browser/package.json]: Okay thanks! I changed to 1.0.0 anyways since we would graduate immediately

**Pull request merged**